5.0/en/0x51-V51-OAuth2.md

```diff
@@ -37,6 +91,8 @@ Given the complexity of the area, it is vitally important for a secure OAuth or
 
 ## V51.4 OAuth Authorization Server
 
+The requirements cover responsibilities for authorization servers, any server in OAuth or OIDC issuing tokens. Depending on context, the authorization server is sometimes also referred to as e. g. OpenID Provider (OP), Identity Provider (IdP), or Security Token Service (STS).
```

Comment:
Since AS and OP is described above, perhaps change

> The requirements cover responsibilities for authorization servers, any server in OAuth or OIDC issuing tokens. Depending on context, the authorization server is sometimes also referred to as e. g. OpenID Provider (OP), Identity Provider (IdP), or Security Token Service (STS).

to

> The requirements cover responsibilities for authorization servers, any server in OAuth or OIDC issuing tokens.

Reply:
Maybe:

> These requirements cover responsibilities for OAuth authorization servers, including OpenID providers.